tests/D2L.CodeStyle.Analyzers.Test.Analyzed/D2L.CodeStyle.Analyzers.Test.Analyzed.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Analyzer Include="../../src/D2L.CodeStyle.Analyzers/bin/Debug/netstandard2.0/D2L.Codestyle.Analyzers.dll" />

```

If this is intended to be included in the main sln, this could be a project reference:

```suggestion
    <ProjectReference Include="../../src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.csproj">
      <OutputItemType>Analyzer</OutputItemType>
      <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
    </ProjectReference>
```

---

No, separate sln, just accidentally deleted it. The project would rather intentionally not compile.